community/urls.py

```diff
@@ -188,18 +188,6 @@ def get_organization():
         name='affiliatedcommitter-detail',
         distill_func=get_all_affiliatedcommitters,
     ),
-    distill_url(
```

Comment:
removing them causes lint errors you should fix before pushing.

and you shouldnt commit stuff like this in your main commit.  it should be a separate commit with a commit summary explaining why it was done.